src/display.ts

```diff
@@ -89,12 +94,13 @@ export const determineAction = async (
 			{ name: `Ignore All Occurrences (${cyan(otherTypoInstancesCount + 1)})`, value: Action.IgnoreAll },
 			{ name: `Replace All Occurrences (${cyan(otherTypoInstancesCount + 1)})`, value: Action.ReplaceAll },
 			{ name: `Skip File (${cyan(otherTyposInFileCount + 1)})`, value: Action.SkipFile },
-			// The explicit undefined check is needed because Action.Ignore is 0, which is falsy.
-			...(previousState.action !== undefined && previousState.action !== Action.UndoLastAction
+			...(shouldDisplayUndoLastAction
+				? [{ name: yellowBright('Undo Last Action'), value: Action.UndoLastAction }]
+				: []),
+			...(shouldDisplayOpenHistory
 				? [
-						{ name: yellowBright('Undo Last Action'), value: Action.UndoLastAction },
 						{
-							name: magentaBright('Open Issue History'),
+							name: magentaBright('Open Action History'),
 							value: Action.OpenHistory
 						}
```

Comment:
Nitpick, could this be formatted on one line like the "undo last action"?

Reply:
Done!